From 00a379018fd7ca9996fcc40ca988db8af1d2ba14 Mon Sep 17 00:00:00 2001
From: Rob Evers <revers@redhat.com>
Date: Mon, 13 Oct 2014 19:47:52 -0400
Subject: [scsi] lpfc: driver release 10.2.8021.1 for RHEL7

Message-id: <1413229672-8452-8-git-send-email-revers@redhat.com>
Patchwork-id: 97524
O-Subject: [RHEL7.1 e-stor PATCH 7/7] scsi: lpfc: driver release 10.2.8021.1 for RHEL7
Bugzilla: 1149846
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Ewan Milne <emilne@redhat.com>

From: Dick Kennedy <rkennedy@emulex.com>

Rhel specific

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/scsi/lpfc/lpfc_version.h b/drivers/scsi/lpfc/lpfc_version.h
index 2f253ab..62d68d9 100644
--- a/drivers/scsi/lpfc/lpfc_version.h
+++ b/drivers/scsi/lpfc/lpfc_version.h
@@ -18,7 +18,7 @@
  * included with this package.                                     *
  *******************************************************************/
 
-#define LPFC_DRIVER_VERSION "10.2.8021.0"
+#define LPFC_DRIVER_VERSION "10.2.8021.1"
 #define LPFC_DRIVER_NAME  "lpfc"
 
 /* Used for SLI 2/3 */
-- 
1.7.1